cloud/amazon/ec2.py

```diff
@@ -613,6 +620,19 @@ def get_instance_info(inst):
         instance_info['ebs_optimized'] = False
 
     try:
+        bdm_dict = {}
```

Comment:
Go ahead and make sure the new feature is fully represented in this PR and update the commit message, and we can merge them all in one shot.